### ISSUE TITLE
Examples incorrectly displayed when generated from OAS

### ISSUE THREAD

**Issue by jebertz** [readmeio/rdme#921]
Our OAS looks something like:
```
  examples:
    markerPaginatedAuditLogsExample:
      summary: 'audit logs example'
      value: {
        'nextMarker': "",
        'previousMarker': "",
        'limit': 20,
        'count': 2,
```
which is getting displayed in readme as:
![Screenshot 2023-10-12 at 2 12 23 PM](https://github.com/readmeio/rdme/assets/57638505/1cbe4289-1c79-4e12-b2af-ac07292920d1)
Note that `summary` and `value` are included in the response as though those were actual fields.  According to the OAS spec, however, what we have is legal OAS (https://swagger.io/specification/#example-object) and I would expect the response object to only show the value of the object associated with `value`

Apologies if this bug belongs somewhere else, I wasn't quite sure where to put it, but hopefully you can help.

**Comment by erunion** [comment]
Yeah your API definition seems to have that set up properly. I'll take a look!

**Comment by erunion** [comment]
@jebertz Are you able to share the OpenAPI definition that was causing this? I'm having trouble replicating with one our testbed definitions[^1] for request examples.

![Screen Shot 2023-10-13 at 9 07 41 AM](https://github.com/readmeio/rdme/assets/33762/1b46feb4-3413-4bd4-8ca8-18b8c711f688)

When rendered in our API Explorer it looks as it should:

![Screen Shot 2023-10-13 at 9 08 22 AM](https://github.com/readmeio/rdme/assets/33762/e5713dc9-ffb0-4453-bb97-2267fc72815e)

[^1]: https://raw.githubusercontent.com/readmeio/oas-examples/main/3.0/json/request-examples.json

**Comment by jebertz** [comment]
Okay, so I made some changes and re-uploaded and am no longer seeing that, so I will assume it was an error on my side. Sorry for the confusion.  

However, after my most recent push, i've noticed that my object is being displayed as : <img width="710" alt="Screenshot 2023-10-13 at 7 15 05 PM" src="https://github.com/readmeio/rdme/assets/57638505/102e4fee-d396-4a7e-a928-9086cc2dd496">
but as far as I can tell it's the exact same format in the OAS as other object I have that are displaying correctly.  Is there something I should be looking for there?  Basically I have 
```
    markerPaginatedAuditLogs:
      type: object
      allOf:
        - $ref: "../pagination.yaml#/components/schemas/markerPaginatedResponse"
        - $ref: "#/components/schemas/auditLogData"
    auditLogData:
      type: object
      required:
        - data
      properties:
        data:
          type: array
          nullable: false
          items:
            $ref: "#/components/schemas/audit-log"
```
where I know that both "#/components/schemas/audit-log" and "../pagination.yaml#/components/schemas/markerPaginatedResponse" are displaying correctly in other endpoints.

If it would be helpful to get our full spec, I messaged you on linkedin since I don't really want to post it all publicly and we have pretty complex refs (meaning you would need a large chunk of our files to fully get everything).
